xml/deployment_installation.xml

```diff
@@ -83,7 +83,239 @@
+          </entry>
+          <entry>
+           <para> Port </para>
+          </entry>
+          <entry>
+           <para> Internal / External </para>
+          </entry>
+          <entry>
+           <para> Open after orchestration </para>
+          </entry>
+          <entry>
+           <para> Description </para>
+          </entry>
+          <entry>
+           <para> Notes </para>
+          </entry>
+         </row>
+        </thead>
+        <tbody>
+         <row>
+          <entry>
+           <para> All machines </para>
+          </entry>
+          <entry>
+           <para> 22 </para>
+          </entry>
+          <entry>
+           <para> Internal </para>
+          </entry>
+          <entry>
+           <para> No </para>
+          </entry>
+          <entry>
+           <para> ssh </para>
+          </entry>
+          <entry>
+           <para> Useful but not essential </para>
+          </entry>
+         </row>
+         <row>
+          <entry morerows="3" valign="middle">
+           <para> Admin </para>
+          </entry>
+          <entry>
+           <para> 80 </para>
+          </entry>
+          <entry>
+           <para> External </para>
+          </entry>
+          <entry>
+           <para> No </para>
+          </entry>
+          <entry>
+           <para> HTTP </para>
+          </entry>
+          <entry/>
+         </row>
+         <row>
+          <entry>
+           <para> 443 </para>
+          </entry>
+          <entry>
+           <para> External </para>
+          </entry>
+          <entry>
+           <para> No </para>
+          </entry>
+          <entry>
+           <para> HTTPS </para>
+          </entry>
+          <entry/>
+         </row>
+         <row>
+          <entry>
+           <para> 2379 </para>
+          </entry>
+          <entry>
+           <para> Internal </para>
+          </entry>
+          <entry>
+           <para> No </para>
+          </entry>
+          <entry>
+           <para>
+            <literal>etcd</literal> discovery </para>
+          </entry>
+          <entry/>
+         </row>
+         <row>
+          <entry>
+           <para> 4505 - 4506 </para>
+          </entry>
+          <entry>
+           <para> Internal </para>
+          </entry>
+          <entry>
+           <para> No </para>
+          </entry>
+          <entry>
+           <para> Salt </para>
+          </entry>
+          <entry/>
+         </row>
+         <row>
+          <entry morerows="2" valign="middle">
+           <para> Masters </para>
+          </entry>
+          <entry>
+           <para> 2380 </para>
+          </entry>
+          <entry>
+           <para> Internal </para>
+          </entry>
+          <entry>
+           <para> Yes </para>
+          </entry>
+          <entry>
+           <para>
+            <literal>etcd</literal> control </para>
+          </entry>
+          <entry>
+           <para> For peer-to-peer etcd traffic </para>
```

Comment:
Is there a reason that etcd does not always have <literal> tags?

Reply:
Line 63: "all nodes should to be"

Reply:
I'm probably using github's review dealie wrong-- line 63 "all nodes should to be"

Reply:
I did not tag ``etcd'' in the Notes column, but it *is* inconsistent. Fixed.

"All nodes should be on the same network" - fixed.

Reply:
Line 222-- etcd again :)

Reply:
I'm not sure that replaceable is the right tag for this-- maybe check with the team tomorrow. Maybe it should be the command tag.

Reply:
I haven't used the <replaceable> tag anywhere, have I?